apps/functions/deadline-submissions/deadline-submission-command/back-office-api-client.js

```diff
@@ -0,0 +1,74 @@
+import got from 'got';
+import config from './config';
+
+/** @typedef {{ id: number, displayNameEn: string }} FolderJSON */
+
+/**
+ * @param {string} caseReference
+ * @returns {Promise<number>}
+ * */
+async function getCaseID(caseReference) {
+	const caseDetails = await got
+		.get(`https://${config.API_HOST}/applications?=${caseReference}`)
+		.json();
+	return caseDetails.id;
+}
+
+/**
+ *
+ * @param {{ caseID: number, documentName: string, documentType: string, documentSize: number, folderID: number, userEmail: string }} _
+ * */
+async function submitDocument({
+	caseID,
+	documentName,
+	documentType,
+	documentSize,
+	folderID,
+	userEmail
+}) {
+	return await got
+		.post(`https://${config.API_HOST}/applications/${caseID}`, {
+			json: {
+				documentName,
+				documentType,
+				documentSize,
+				folderId: folderID,
+				username: userEmail
+			}
+		})
+		.json();
+}
+
+/**
+ *
+ * @param {number} caseID
+ * @param {string} caseName
+ * @param {string} lineItem
+ * @returns {Promise<number>}
+ * */
+async function getFolderID(caseID, caseName, lineItem) {
```

Comment:
It could be tricky since line items aren't in a table of their own, they're just in a JSON structure in the ExaminationTimetableItem table. Are you thinking the folder ID would be stored in the JSON structure with the line items?